### ISSUE TITLE
Random movement, crouching, swimming and POV flashing.

### ISSUE THREAD

**Issue by SteaSteaStea** [Engine-Room/Flywheel#19]
When using this mod (with or without Create) there appears to be this strange issue where you character randomly moves from side to side / crouches /enters the swimming animation for a split second. Another thing happening is the POV glitching out randomly (it's most apparent in 3rd person mode, in 1st person it looks like an occasional black screen flash). It seems to get worse if you increase the render distance or in general put more strain on the system.

It seems to be a compatibility problem. A clean instance with only forge+flywheel doesn't generate this issue.
I will try to scan through the mods to see what exactly causes this.

**Comment by OHKOBunny** [comment]
Same thing's happening to me, I tracked the root issue to Flywheel but it seems to be a bizarre incompatibility with Optifine and Immersive Engineering's fluid pipes. Behavior also occurs to a significantly lesser degree with Tinker's Construct's fluid transfer thingies.

If you're using any of these mods,  it's them. I have noticed that it doesn't happen when not using OF and the behavior only seems to trigger when attempting to render the fluid pipes, or in some cases, pumps. 

Also worth noting that this all started happening (to me at least) after the last forge update. I hadn't seen it happen prior to that.

tldr; I only notice it happening when using Flywheel + OF and looking at the direction of fluid pipes from Immersive Engineering

It also crashes me sometimes

**Comment by Jozufozu** [comment]
Could you upload the crash report you got to [pastebin](https://pastebin.com/)?

**Comment by OHKOBunny** [comment]
> Could you upload the crash report you got to [pastebin](https://pastebin.com/)?

https://pastebin.com/Y5ZA6c95

**Comment by OHKOBunny** [comment]
Bonus crashlog fresh after setting up the Press from and peeping the JEI for the crusher:
https://pastebin.com/prDT2P4W

**Comment by SteaSteaStea** [comment]
It happens to me even without OptiFine, and in freshly generated terrain without any contraptions/farms/etc.
Also forgot to mention that I'm running an AMD GPU (RX550).

**Comment by ghost** [comment]
I have this problem too and with chunks glitches, i wrote [here](https://github.com/Jozufozu/Flywheel/issues/18) about this

**Comment by Jozufozu** [comment]
Just released [a patch](https://www.curseforge.com/minecraft/mc-mods/flywheel/files/3411101), are you still having these issues?

**Comment by SteaSteaStea** [comment]
The issue is gone. Thanks for the quick patch.
